support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -4,7 +4,6 @@ import type { Tests } from './abtest';
 // ----- Tests ----- //
 
 const allLandingPagesAndThankyouPages = '/contribute|thankyou(/.*)?$';
-const allThankYouPages = '/thankyou(/.*)?$';
```

Comment:
Perhaps it's worth having a comment here in the code that mentions that we shouldn't target just the TYP for these tests?

Reply:
👍